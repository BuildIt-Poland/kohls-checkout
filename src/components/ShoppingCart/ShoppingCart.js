import React from 'react';
import styled from 'styled-components';

import { spacingMedium } from '../../styles/designTokens';
import ItemsList from '../ItemsList';
import SectionHeader from '../SectionHeader';

const Wrapper = styled.div`
  padding-left: ${spacingMedium};
  padding-right: ${spacingMedium};
`;

function ShoppingCart() {
  return (
    <>
      <SectionHeader title="Shopping Cart"> </SectionHeader>
      <Wrapper>
        <ItemsList />
      </Wrapper>
    </>
  );
}

export default ShoppingCart;
