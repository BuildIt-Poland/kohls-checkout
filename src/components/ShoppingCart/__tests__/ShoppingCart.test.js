import React from 'react';
import { create } from 'react-test-renderer';

import ShoppingCart from '../ShoppingCart';

jest.mock('../../ItemsList', () => 'ItemsList');

describe('COMPONENT - ShoppingCart', () => {
  it('renders ShoppingCart component', () => {
    const component = create(<ShoppingCart />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
