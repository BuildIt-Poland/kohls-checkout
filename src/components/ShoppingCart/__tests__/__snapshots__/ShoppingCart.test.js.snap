// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - ShoppingCart renders ShoppingCart component 1`] = `
Array [
  .c0 {
  font-size: 1.2rem;
  text-align: left;
  margin-top: 10px;
  margin-bottom: 10px;
  text-transform: uppercase;
}

<h3
    className="c0"
    title="Shopping Cart"
  >
     
  </h3>,
  .c0 {
  padding-left: 2rem;
  padding-right: 2rem;
}

<div
    className="c0"
  >
    <ItemsList />
  </div>,
]
`;
